checks/testdata/Dockerfile-not-pinned-as

```diff
@@ -0,0 +1,26 @@
+
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Note: taken from https://github.com/pushiqiang/utils/blob/master/docker/Dockerfile_template
+FROM python:3.7@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2 as base
+
+FROM python:3.7 as build
+ARG TARGETOS
+ARG TARGETARCH
+RUN CGO_ENABLED=0 make build-scorecard
+
+FROM build
+RUN /hello-world
+
+FROM python@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2
```

Comment:
remove sha, test should be failing.

Reply:
right. My thinking was there is at least one line in the file that should fail, not necessarily all of them. Shall I change them all or keep it as is?

Reply:
My bad, didnt see the other lines.